libgadget/utils/mpsort.c

```diff
@@ -48,8 +48,8 @@ DEFTYPE(uint64_t)
 static int _compar_radix(const void * r1, const void * r2, size_t rsize, int dir) {
     size_t i;
     /* from most significant */
-    const unsigned char * u1 = r1;
-    const unsigned char * u2 = r2;
+    const unsigned char * u1 = (const unsigned char *) r1;
```

Comment:
We'll need to sync the changes to mpsort.

Reply:
I can open a PR if you would like them

Reply:
Yes. That would help. Thanks!